### DISCUSSION TITLE
How to scroll to a specific tab?

### DISCUSSION THREAD

**Discussion by vhugogarcia** [roubachof/Sharpnado.Tabs#50] [Q&A]
Thank you so much for this great library! it helped me a lot to build quality apps. 

However, I am facing a minor issue, would like to visually scroll to a selected tab programmatically. Basically, I have a quick direct link button in the screen and I want that if the user tabs it, it selects the tab, displays the content but also the horizontal scroll-able tabs positioning correctly in the screen.

I was able to select the tab and display the content correctly, however, the scrollable tabs remain on the same position, I need to find a way to scroll To a position or item. Is that possible somehow with the TabHost or something?

Thanks in advance

**Comment by needledrag** [comment]
Hiya, did you manage to  get this to work, facing  similar requirements.

Cheers
Ahaun

**Reply by vhugogarcia** [reply]
I did not. We may need to add somehow this functionality.
